DateTime.js

```diff
@@ -169,6 +169,10 @@ var Datetime = React.createClass({
 				update.open = this.state.open;
 			}
 		}
+        
+        if ( nextProps.viewMode !== this.props.viewMode ) {
```

Comment:
The indentation of this whole block is not correct either. I know it looks correct in your editor, but it never looks good on GH. I'm not sure why but I will try to come up with a solution for this..

I usually copy whitespace from other parts of the file and paste, that usually works. And then double checks when I have pushed to GH.

Anyway, this should be fixed for this PR.